### PR TITLE
Bump clustertest to v0.3.0

### PR DESCRIPTION
**Pull request by fiunchinho**
### What this PR does
Bump clustertest to v0.3.0 to be able to set app organization.

### Checklist

- [X] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bumped clustertest to `v0.3.0` to be able to set the app `Organization`.
```
Comment (fiunchinho) [inline CHANGELOG.md:12]: I'll fix this after tests pass :cry:

File: common/dns.go [hunk 2]
```diff
@@ -6,12 +6,13 @@ import (
...
 	"time"
 
-	"github.com/giantswarm/cluster-test-suites/internal/state"
 	"github.com/giantswarm/clustertest/pkg/application"
 	"github.com/giantswarm/clustertest/pkg/logger"
 	"github.com/giantswarm/clustertest/pkg/wait"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	"github.com/giantswarm/cluster-test-suites/internal/state"
```
Comment (fiunchinho) [inline common/dns.go:15]: This is aligned with [what we described here](https://github.com/giantswarm/fmt/blob/master/go/imports.md).

**Review by AverageMarcus — APPROVED** [review]

**Comment by AverageMarcus** [comment]
/run cluster-test-suites

(If its just the CAPV failure I think we can push this through)

**Pull request merged**